src/components/TextField/TextField.tsx

```diff
@@ -48,14 +48,25 @@ export const useStyles = makeStyles(
         opacity: 0.9,
       },
     },
+    inputContainer: {
+      position: 'relative',
+      minWidth: theme.pxToRem(180),
+      width: 'fit-content',
+    },
+    inputContainerFullWidth: {
+      width: '100%',
+      '& $inputHasAdornment': {
+        maxWidth: 'unset',
+      },
+    },
     input: {
       backgroundColor: 'rgba(132, 137, 166, 0.15)',
       border: `1px solid transparent`,
       borderRadius: theme.pxToRem(4),
       color: theme.palette.text.primary,
       fontFamily: theme.typography.fontFamily,
       fontSize: theme.pxToRem(14),
-      minWidth: theme.pxToRem(175),
+      minWidth: theme.pxToRem(180),
```

Comment:
Not sure if this qualifies as a breaking change? I needed to bump the `minWidth` of `<TextField` to get consistent widths across Chrome, Firefox and Safari. This could impact some layouts?

Reply:
Hm I probably wouldn't consider it a breaking change since it's only 5 pixels and there weren't breaking API changes; however, I'm curious what other folks think